driver/normalizer/annotation.go

```diff
@@ -236,10 +247,27 @@ var AnnotationRules = On(Any).Self(
 		On(HasInternalType(pyast.Return)).Roles(Return),
 		On(HasInternalType(pyast.Break)).Roles(Break),
 		On(HasInternalType(pyast.Continue)).Roles(Continue),
+		// FIXME: IfCondition bodies in Python take the form:
+		// 1 < a < 10
+		// - left (internalRole): 1 (first element)
+		// - Compare.ops (internalType): [LessThan, LessThan]
+		// - Compare.comparators (internalType): ['a', 10]
+		// The current mapping is:
+		// - left: BinaryExpressionLeft
+		// - Compare.ops: BinaryExpressionOp
+		// - Compare.comparators: BinaryExpressionRight
```

Comment:
Mmm it's pretty common to call left and right the elements of a binary expression (just like lvalue and rvalue on assignments), they could be inferred by the order or called first/second.

Your tough about generalizing UnaryExpressions, BinaryExpressions, etc to NExpressions or something like that makes a lot of sense.